bin/templates/scripts/cordova/lib/plugman/pluginHandlers.js

```diff
@@ -66,17 +66,19 @@ var handlers = {
         install:function(obj, plugin, project, options) {
             var src = obj.src,
                 custom = obj.custom;
-
             if (!custom) {
                 var keepFrameworks = keep_these_frameworks;
 
-                if (keepFrameworks.indexOf(src) < 0) {
-                    project.xcode.addFramework(src, {weak: obj.weak});
-                    project.frameworks[src] = (project.frameworks[src] || 0) + 1;
+                if (keepFrameworks.indexOf(src) < 0) { 
+                    if (obj.type === 'podspec') {
+                        //podspec handled in Api.js
+                    } else {
+                        project.frameworks[src] = (project.frameworks[src] || 0) + 1;
```

Comment:
not your code, but dislike code like this that tries to be clever although I know the meaning is "increment by 1. set to zero first if it doesn't exist"